agent/consul/stats_fetcher.go

```diff
@@ -5,11 +5,12 @@ import (
 	"net"
 	"sync"
 
-	"github.com/hashicorp/consul/agent/pool"
-	"github.com/hashicorp/consul/agent/structs"
 	"github.com/hashicorp/go-hclog"
 	"github.com/hashicorp/raft"
 	autopilot "github.com/hashicorp/raft-autopilot"
+
+	"github.com/hashicorp/consul/agent/pool"
```

Comment:
this is changed per https://docs.google.com/document/d/1uhyo1O2Uzp_o9OvdAjXiYSvmr56M3ZsMGp9v9nD72Vw/edit#heading=h.9xjgdxm5l2qp